### PR TITLE
Feature/somewhat improved frontend

### PR DESCRIPTION
**Pull request by x183**
is bootiful :rose:

### CODE REVIEW

**Review by Supergamer1337 — CHANGES_REQUESTED** [review]
Some questions, and a suggestion

File: frontend/components/Search/Search.tsx [hunk 1]
```diff
@@ -26,7 +26,10 @@ const searchHook = (searchValue: string) => {
           headers: {
             "Content-Type": "application/json",
           },
-          body: JSON.stringify({ name: searchValue, amount: 10 }),
+          body: JSON.stringify({
+            name: searchValue,
+            amount: 20 /* document.body.clientHeight / 50 */,
```
[the remark this answers is not in the document]
Comment (x183) [reply frontend/components/Search/Search.tsx]: No

[the remark this answers is not in the document]
Comment (Supergamer1337) [reply frontend/components/Search/Search.tsx]: Leave a comment explaining why that is then

**Review by Supergamer1337 — APPROVED** [review]
LGTM

**Pull request merged**